Blocks/HTTP/request.h

```diff
@@ -93,25 +107,25 @@ struct Request final {
   const current::net::HTTPRequestData&
       http_data;  // Accessor to use `r.http_data` instead of `r.connection->HTTPRequest()`.
   const URL url;
-  const URLPathParams url_path_params;
+  const URLPathArgs url_path_args;
   const std::string method;
   const current::net::HTTPRequestData::HeadersType& headers;
   const std::string& body;  // TODO(dkorolev): This is inefficient, but will do.
   const std::chrono::microseconds timestamp;
 
   explicit Request(std::unique_ptr<current::net::HTTPServerConnection>&& connection,
-                   URLPathParams url_path_params = URLPathParams())
+                   URLPathArgs url_path_args = URLPathArgs())
       : unique_connection(std::move(connection)),
         connection(*unique_connection.get()),
         http_data(unique_connection->HTTPRequest()),
         url(http_data.URL()),
-        url_path_params(url_path_params),
+        url_path_args(url_path_args),
         method(http_data.Method()),
         headers(http_data.headers()),
         body(http_data.Body()),
         timestamp(current::time::Now()) {
-    if (!url_path_params.empty()) {
-      url.path.resize(url_path_params.base_path.length());
+    if (!url_path_args.empty()) {
+      url.path.resize(url_path_args.base_path.length());
```

Comment:
Truncating `url.path` in any case, not depending on `url_path_args` will help to fix trailing slashes problem.